<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Tablet Tester (%1)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title">
      <property name="text">
       <string>&lt;h1&gt;Qt Tablet Tester&lt;/h1&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select a mode below to get started:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pressureLevelButton">
      <property name="text">
       <string>Pressure Level Tester</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tiltButton">
      <property name="text">
       <string>Stylus Tilt Tester</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reportRateButton">
      <property name="text">
       <string>Report Rate Tester</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rotationButton">
      <property name="text">
       <string>Stylus Rotation Tracker</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="eventLoggerButton">
      <property name="text">
       <string>Event Logger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="freeDrawButton">
      <property name="text">
       <string>Free Draw</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
